lib/runtime/Cargo.toml

```diff
@@ -77,5 +77,13 @@ socket2 = { version = "0.5.8" }
 [dev-dependencies]
 assert_matches = { version = "1.5.0" }
 env_logger = { version = "0.11" }
+reqwest = { version = "0.12.22", features = ["json"] }
```

Comment:
I'm trying to avoid us having a dependency on OpenSSL. Fix is here: https://github.com/ai-dynamo/dynamo/pull/1945